### ISSUE TITLE
Failed to execute installation Apple Script

### ISSUE THREAD

**Issue by idelice** [alex35mil/NeoHub#6]
## What happened?
_Reproduction steps, context, etc._

## Error details
```
Failed to execute installation Apple Script
App: Version 0.1.1 (Build 7057133)
macOS: Version 14.1.2 (Build 23B92)
Arch: arm64
Context: InstallationView#body
Metadata:
    ["NSAppleScriptErrorAppName": NeoHub, "NSAppleScriptErrorMessage": cp: /usr/local/lib/NeoHubLib.framework: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework: No such file or directory
cp: symlink: Versions/Current/NeoHubLib: No such file or directory
cp: symlink: Versions/Current/Resources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/NeoHubLib: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A/_CodeSignature: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature/CodeResources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/Resources: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A/Resources: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A/Resources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/Resources/Info.plist: No such file or directory
cp: symlink: A: No such file or directory, "NSAppleScriptErrorRange": NSRange: {0, 438}, "NSAppleScriptErrorBriefMessage": cp: /usr/local/lib/NeoHubLib.framework: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework: No such file or directory
cp: symlink: Versions/Current/NeoHubLib: No such file or directory
cp: symlink: Versions/Current/Resources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/NeoHubLib: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A/_CodeSignature: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature/CodeResources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/Resources: No such file or directory
cp: /private/var/folders/7x/dkdc29wx6yl7tf0gvf7wrbqh0000gn/T/AppTranslocation/A6A77CAC-BAF3-428C-A9DE-21B310B8106F/d/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A/Resources: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A/Resources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/Resources/Info.plist: No such file or directory
cp: symlink: A: No such file or directory, "NSAppleScriptErrorNumber": 1]
```

**Comment by alex35mil** [comment]
It should be fixed by #8.

**Comment by alex35mil** [comment]
It should be fixed in [`0.2.1`](https://github.com/alex35mil/NeoHub/releases/tag/v0.2.1`), but please see #15.

**Comment by alex35mil** [comment]
FYI I just updated `neovim` to the latest stable and `neovide` to the latest master (`0.13.1`) and things work again with `neohub@0.2.1`.
